docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+
+## [Additional features: Observables](@id petab_observables)
+
+### [Defining non-trivial observables](@id petab_observables_observables)
+It is possible for observables to be any algebraic expression of species concentrations and parameters. E.g. in this example the total amount of `X` in the system is an observable:
+```petab2
+using Catalyst, PEtab # hide
+rn = @reaction_network begin
+    (k1,k2), X1 <--> X2
+end
+@unpack X1, X2 = rn
+obs_X = PEtabObservable(X1 + X2, 0.5)
+```
+
+### [Advanced observables noise formulas](@id petab_observables_noise_formula)
+In our basic example we assumed that the normally distributed noise had a standard deviation of `0.5`. However, this value may be a parameter (or indeed any algebraic expression). E.g, we could set
+```petab1
+@parameters σ
+obs_P = PEtabObservable(P, σ)
+```
+and then let the parameter *σ* vary between different [simulation conditions](@ref petab_simulation_conditions). Alternatively we could let the noise scale linearly with the species' concentration:
+```petab1
+obs_P = PEtabObservable(P, 0.05P)
+```
+
+PEtab.jl assumes that noise is normally distributed (with a standard deviation equal to the second argument provided to `PEtabObservable`). The only other (currently implemented) noise distribution is log-normally distributed noise, which is designated through the `transformation=:log` argument:
+```petab1
+obs_P = PEtabObservable(P, σ; transformation=:log)
+```
+
+## [Additional features: Parameters](@id petab_parameters)
+
+### [Known parameters](@id petab_parameters_known)
+In our previous example, all parameters were unknowns that we wished to fit to the data. If any parameters have known values, it is possible to provide these to `PEtabModel` through the `parameter_map` argument. E.g if we had known that *kB = 1.0*, then we would only define *kD* and *kP* as parameters we wish to fit:
+```petab1
+par_kD = PEtabParameter(:kD)
+par_kP = PEtabParameter(:kP)
+params = [par_kD, par_kP]
+nothing # hide
+```
+We then provide `parameter_map=[:kB => 1.0]` when we assembly our model:
+```petab1
+petab_model = PEtabModel(rn, observables, measurements, params; state_map=u0, parameter_map=[:kB => 1.0])
+nothing # hide
+```
+
+### [Parameter bounds](@id petab_parameters_bounds)
+By default, when fitted, potential parameter values are assumed to be in the interval *[1e-3, 1e3]*. When declaring a `PEtabParameter` it is possible to change these values through the `lb` and `ub` arguments. E.g. we could use
+```petab1
+par_kB = PEtabParameter(:kB; lb=1e-2, ub=1e-2)
+```
+to achieve the more conservative bound *[1e-2, 1e2]* for the parameter *kB*.
+
+### [Parameter scales](@id petab_parameters_scales)
+
+Internally, parameters that are fitted are converted to a logarithmic scale (generally, this is considered advantageous). To prevent this conversion, the `scale=:lin` argument can be used. Here we change the scale of *kB* to linear:
```

Comment:
Do we want to add citation here, as it is proven typically better in this paper  https://academic.oup.com/bioinformatics/article/35/17/3073/5280731?login=true

Reply:
might as well, will add.

Reply:
Added. Generally, I like the idea of having 1-4 useful references on the longer do pages. Not more to make it like a research article, but more to give some useful further readings to the user.